src/monster.cpp

```diff
@@ -430,6 +432,12 @@ int monster::print_info(WINDOW* w, int vStart, int vLines, int column) const
     const auto hp_desc = hp_description( hp, type->hp );
     mvwprintz( w, vStart++, column, hp_desc.second, "%s", hp_desc.first.c_str() );
 
+    if (!inv.empty()) {
+        // Cut off really long item names
+        std::vector<std::string> lines = foldstring( inventory_summary(), getmaxx(w) - 1 - column );
+        mvwprintz( w, vStart++, column, c_yellow, "Carrying %s.", lines.front().c_str() );
```

Comment:
This is not translated. Add `_( )` around *all* string literals that should be translated. Applies also to code in `inventory_summary` and in `extended_description`.